### PR TITLE
Fix EZP-30857: Referenced image file is not deleted from storage when removing draft, archived or published version.

### PR DESCRIPTION
**Pull request by blankse**
Fix https://issues.ibexa.co/browse/EZP-30857 also for legacy backend

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -31,6 +31,7 @@ before_script:
   - phpenv config-rm xdebug.ini
   - echo "memory_limit=-1" >> ~/.phpenv/versions/$(phpenv version-name)/etc/conf.d/travis.ini
   # Install packages using composer
+  - composer self-update --1
```
Comment (andrerom) [inline .travis.yml:34]: Note to self: We need separate PR to get CI working with composer 2

Comment (blankse) [reply .travis.yml:34]: I think it should be working if this is tagged https://github.com/ezsystems/ezpublish-kernel/commit/2e438043b7b1e644b8467bd9d84f9a2347625867

Comment (andrerom) [reply .travis.yml:34]: Ok, that should afaik happen soon

Comment (blankse) [reply .travis.yml]: Now it works with composer 2

**Comment by blankse** [comment]
I needed to upgrade php-cs-fixer because of https://github.com/FriendsOfPHP/PHP-CS-Fixer/pull/4905

**Comment by andrerom** [comment]
@blankse We should do CS fixes separately from this PR, I'll look into applying c3e63b93a6c2a66237d6681b0ac8f7ba74a330a2 on maintained branches and let you know.

**Comment by blankse** [comment]
@andrerom Should I create a PR for it? For which branch?

**Comment by blankse** [comment]
I opened https://github.com/ezsystems/LegacyBridge/pull/180

**Comment by blankse** [comment]
I rebased because of merging #180

**Comment by emodric** [comment]
Looks good :+1:

**Review by emodric — APPROVED** [review]

**Review by konradoboza — APPROVED** [review]

**Review by tomaszszopinski — APPROVED** [review]
QA approved on eZPlatform-ee 2.5 (legacy installed) with patch.

**Comment by konradoboza** [comment]
Thank you @blankse!

**Pull request merged**